package/MDAnalysis/topology/MMTFParser.py

```diff
@@ -0,0 +1,220 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver Beckstein
+# and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+"""
+MMTF Topology Parser
+====================
+
+Reads topology data from the `Macromolecular Transmission Format
+(MMTF) format`_.  This should generally be a quicker alternative to PDB.
+
+Makes individual models within the MMTF file available via the `models`
+attribute on Universe.
```

Comment:
Can you explain a little bit more what `model` means (i.e., different topologies are possible) and how one does that? This is new functionality for MDAnalysis and it appears at `Universe.model` so this should be fully documented, ideally with an example (from the tests?).